SKAToolKit/SKATiledMap/SKATiledMap.m

```diff
@@ -772,20 +762,31 @@ - (void)cullAroundIndexX:(NSInteger)x
             endingY = 0;
         }
 
+        NSMutableArray *spritesToTrash = [[NSMutableArray alloc] init]; //array to hold sprites that have moved off-screen
+        [self.visibleArray
+            enumerateObjectsUsingBlock:^(id obj, NSUInteger idx, BOOL *stop) {
```

Comment:
Instead of enumerating every node in `visibleArray`, is it possible to just grab all the nodes in the far left and far right columns and check if they’ve moved off screen? I'm thinking it would be possible, and would eliminate a lot of unnecessary looping